portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,105 @@
+body {
+    background-color: black;
+    color: white;
+    margin-left: 40px;
+    margin-right: 20px;
+}
+
+h1{
```

Comment:
nit: 1 space after h1. 
h1 { instead of h1{

Reply:
resolved